<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmPickDialog</class>
 <widget class="QDialog" name="AlarmPickDialog">
  <property name="windowTitle">
   <string>Alarm</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
  </property>
  <layout class="QGridLayout" name="dialogLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QCheckBox" name="enableBox">
       <property name="text">
        <string>Enable</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="configWidget">
       <layout class="QVBoxLayout" name="configLayout">
        <item>
         <layout class="QHBoxLayout" name="typeLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="beforeButton">
            <property name="text">
             <string>In advance</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="triggerButton">
            <property name="text">
             <string>Arbitrary</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="beforeWidget">
          <layout class="QGridLayout" name="beforeLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="hoursLabel">
             <property name="text">
              <string>Hours</string>
             </property>
             <property name="indent">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ChainedSpinBox" name="hoursBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>168</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="minutesLabel">
             <property name="text">
              <string>Minutes</string>
             </property>
             <property name="indent">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ChainedSpinBox" name="minutesBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="secondsLabel">
             <property name="text">
              <string>Seconds</string>
             </property>
             <property name="indent">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ChainedSpinBox" name="secondsBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="triggerWidget">
          <layout class="QHBoxLayout" name="triggerLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QMaemo5ValueButton" name="dateButton">
             <property name="valueLayout">
              <enum>QMaemo5ValueButton::ValueBesideText</enum>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QMaemo5ValueButton" name="timeButton">
             <property name="valueLayout">
              <enum>QMaemo5ValueButton::ValueBesideText</enum>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChainedSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ChainedSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
